frontend/src/Pages/Admin.js

```diff
@@ -22,7 +22,7 @@ const Admin = () => {
   if (isLoading) return "Loading...";
 
   // we need to change this error!
-  if (!data?.[0] || error) {
+  if (!data?.[0] && error) {
```

Comment:
I think it's safe to assume here that if `error` is truthy then `data` will not contain anything? Axios should only return an error if the call failed.

```suggestion
  if (error) {
```